targets/TARGET_NORDIC/TARGET_MCU_NRF51822/i2c_api.c

```diff
@@ -310,20 +344,20 @@ int i2c_byte_write(i2c_t *obj, int data)
 
 const PinMap *i2c_master_sda_pinmap()
 {
-    return PinMap_I2C_SDA;
+    return PinMap_I2C_testing;
```

Comment:
The comment says this pinmap is for testing only. Are `i2c_master_sda_pinmap()` and other `i2c_*_pinmap()` functions intended to be used only in tests?

Reply:
Yep, all the I2C pinmaps on the NRF5X are for testing only. The actual hardware is fully muxable so you can use any peripheral on any pin. Because of this the pinmap isn't needed by the NRF5X HAL for for finding a mapping between a given pin and a peripheral. This is only useful to know what to test from CI.